spi_nand_flash/src/spi_nand_oper.c

```diff
@@ -26,7 +27,17 @@
         .dummy_bits = transaction->dummy_bits
     };
 
-    return spi_device_transmit(device, (spi_transaction_t *) &e);
+    if (transaction->flags == SPI_TRANS_USE_TXDATA && transaction->mosi_len <= 4) {
+        memcpy(e.base.tx_data, transaction->mosi_data, transaction->mosi_len);
+    }
+
+    esp_err_t ret = spi_device_transmit(device, (spi_transaction_t *) &e);
+    if (ret == ESP_OK) {
+        if (transaction->flags == SPI_TRANS_USE_RXDATA && transaction->miso_len <= 4) {
+            memcpy(transaction->miso_data, e.base.rx_data, transaction->miso_len);
```

Comment:
@igrr can I dismiss this alert with won't fix reason?

Reply:
Yes, you can.

I'll see if we can somehow filter our `memcpy` warnings in clang-tidy, since we don't have `memcpy_s` in newlib yet. I don't want to disable `DeprecatedOrUnsafeBufferHandling` completely because it is still useful for other functions like `strcpy`.